src/renderer/Default.js

```diff
@@ -220,6 +217,9 @@ export default function Default() {
                           render={props => <MarketCoinScreen {...props} />}
                         />
                         <Route path="/market" render={props => <Market {...props} />} />
+                        <FeatureToggle feature="learn">
+                          <Route path="/learn" render={props => <Learn {...props} />} />
+                        </FeatureToggle>
```

Comment:
is this supposed to be in this PR?

Reply:
it fixes a regression on feature flag :+1: